frontend/src/profiles/ProfileDetails.js

```diff
@@ -27,10 +26,10 @@ export default function ProfileDetails(){
             <ProfileDetailsStyled>
                 <div>
```

Comment:
Is the outer `<div>` really needed? Try to keep your hierachy as flat as possible.